apps/docs/content/guides/ai/examples/semantic-image-search-amazon-titan.mdx

```diff
@@ -0,0 +1,228 @@
+---
+id: 'semantic-image-search-python-amazon-titan'
+title: 'Semantic Image Search with Amazon Titan'
+description: 'Implement semantic image search with Amazon Titan and Supabase Vector in Python.'
+subtitle: 'Implement semantic image search with Amazon Titan and Supabase Vector in Python.'
+tocVideo: 'A3uND5sgiO0'
+---
+
+[Amazon Bedrock](https://aws.amazon.com/bedrock) is a fully managed service that offers a choice of high-performing foundation models (FMs) from leading AI companies like AI21 Labs, Anthropic, Cohere, Meta, Mistral AI, Stability AI, and Amazon via a single API, along with a broad set of capabilities you need to build generative AI applications with security, privacy, and responsible AI.
+
+[Amazon Titan](https://aws.amazon.com/bedrock/titan/) is a family of foundation models (FMs) for text and image generation, summarization, classification, open-ended Q&A, information extraction, and text or image search.
+
+In this guide we'll look at how we can get started with Amazon Bedrock and Supabase Vector in Python using the Amazon Titan multimodal model and the [vecs client](/guides/ai/vecs-python-client).
+
+You can find the full application code as a Python Poetry project on [GitHub](https://github.com/supabase/supabase/tree/master/examples/ai/aws_bedrock_image_search).
```

Comment:
this link currently 404s

Reply:
That's okay, the example is part of the PR.